src/zebras/_period.py

```diff
@@ -339,7 +339,21 @@ def _encode_zebras_periods(v: Periods):
     }
 
 
-def period(data, freq=None) -> Period:
+def period(data: Union[Period, str], freq: Optional[str] = None) -> Period:
```

Comment:
@jaheba Still not sure what to do here. Should we import a pandas? Or modify the code to allow using zebras.Period as input too.

Reply:
I think we should allow ``zebras.Period`` as well. Can we check if ``pandas.Period`` has some common base class that we could use here?

Otherwise, we just leave it undocumented for now, I think.

Reply:
I think we should also allow to pass `zebras.Freq` as `freq`.